5_exercicio.js

```diff
@@ -0,0 +1,22 @@
+/*Crie uma função que receba dois números e retorne se o primeiro é maior ou igual ao segundo.
+Exemplos:
+maiorOuIgual(0, 0) // retornará true
+maiorOuIgual(0, "0") // retornará false
+maiorOuIgual(5, 1) // retornará false
+*/
+
+const maiorOuIgual = (a, b) => {
+    if (a > b) {
```

Comment:
Seria bom converter 'a' e 'b' para inteiro

Reply:
Não precisa converter para inteiro. Pois quando for string é para retornar false.

Reply:
Ok